ChangeLog

```diff
@@ -12,6 +12,8 @@ What's New in astroid 2.8.6?
 ============================
 Release date: TBA
 
+* Add a common caching mechanism (`astroid.cache.LRUCache`) and a global
```

Comment:
Due to the review taking a little longer this will need to be moved to `2.10`.

Reply:
I've moved the change log to 2.10.0.